src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final Properties DB_PROPERTIES = new Properties();
+
+    static {
+        try (InputStream input = ConnectionUtil.class.getClassLoader()
+                .getResourceAsStream("db/db.properties")) {
+            if (input == null) {
+                throw new RuntimeException("Sorry, unable to find db.properties in db package");
```

Comment:
Instead of throwing a generic `RuntimeException` here, consider creating a custom exception that more accurately reflects the nature of the error, such as `ConfigurationException` or `PropertiesLoadException`. This would make it clearer to the caller what kind of error occurred and would be more informative for debugging purposes.